### PR TITLE
HttpClientBuilderExtensions.AddJsonClient: add support usage optional parameter `clientName`.

### PR DESCRIPTION
**Pull request by NYMEZIDE**
tests: HttpClientBuilderExtensionsTests: add 2 test case on new features

Issue #25

### CODE REVIEW

**Comment by Ceridan** [comment]
I will double-check compare versions workflow today.

File: src/Dodo.HttpClient.ResiliencePolicies/HttpClientBuilderExtensions.cs [hunk 1]
```diff
@@ -30,13 +30,20 @@ public static class HttpClientBuilderExtensions
...
+			Action<System.Net.Http.HttpClient> defaultClient = (client) => 
+			{
+				client.BaseAddress = baseAddress;
+				client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+				client.Timeout = settings.HttpClientTimeout;
+			};
+
+			IHttpClientBuilder httpClientBuilder;
+			if (!string.IsNullOrEmpty(clientName))
+				httpClientBuilder = sc.AddHttpClient<TClientInterface, TClientImplementation>(clientName, defaultClient);
```
Comment (Ceridan) [inline src/Dodo.HttpClient.ResiliencePolicies/HttpClientBuilderExtensions.cs:42]: Please, could you add curly brackets for the single-line if-statements. I know, there are lack of style guide for this, but we are usually prefer to use it for any if-statement body for consistency.

Also, I ask you to add it to check my changes to the PR-check workflow. I will merge your PR anyway, but lets give it one more try.

**Review by Ceridan — CHANGES_REQUESTED** [review]
Great job! Thank you very much for adding this functionality and write tests for it. Your contribution is very appreciated!

File: src/Dodo.HttpClient.ResiliencePolicies/HttpClientBuilderExtensions.cs (hunk 1 above)
Comment (NYMEZIDE) [reply src/Dodo.HttpClient.ResiliencePolicies/HttpClientBuilderExtensions.cs]: did it.

**Review by Ceridan — APPROVED** [review]

**Comment by Ceridan** [comment]
Thank you very much! Great job!

**Pull request merged**